plugins/commander.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: commander
+spec:
+  homepage: https://github.com/schabrolles/kubectl-commander.git
+  shortDescription: Interactively manage Kubernetes resources.
+  version: "v0.4.0"
+  description: |
+    kube-commander is an interactive menu (based on fzf) that allows you
+    to navigate and interact with objects of your Kubernetes cluster.
+    It is possible to quickly preview object configuration in YAML,
+    logs, events but also edite or delete object or enter into pods.
+
+    [ctrl-h] inside the plugin to get the list of available key bindings.
+  caveats: |
+    For optimal experience, be sure to have the following binaries
+    installed on your machine:
+    fzf:  https://github.com/junegunn/fzf/releases
+    yq:  https://github.com/mikefarah/yq/releases
+  platforms:
+    - selector:
+        matchExpressions:
+          - key: os
+            operator: In
+            values:
+              - darwin
+              - linux
+      uri: https://github.com/schabrolles/kubectl-commander/archive/refs/tags/V0.4.0.tar.gz
```

Comment:
I recommend using lowercase `v` like other plugins do. I believe the auto-approver bot we have only automatically approves PRs if `newURL=replace(oldURL, oldVer,newVer)` (and `version:` field has lowercase `v`).

Reply:
Agree, I regenerated v0.4.0 release from github, because of a previous v0.4.0 existed, the uppercase was added automatically (even if I deleted the old one before).

the next release will have lowercase v